float32 rub
---
float32 usd
float32 eur
float32 gbr
float32 ugx
